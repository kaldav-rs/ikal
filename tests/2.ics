BEGIN:VCALENDAR
VERSION:2.0
PRODID:ownCloud Calendar
CALSCALE:GREGORIAN
BEGIN:VEVENT
UID:7e27d9fc69
DTSTAMP:20150717T101959Z
CREATED:20150717T101959Z
LAST-MODIFIED:20150717T101959Z
SUMMARY:Garage
DTSTART;VALUE=DATE:20150722
DTEND;VALUE=DATE:20150723
LOCATION:
DESCRIPTION:
CATEGORIES:
END:VEVENT
END:VCALENDAR
