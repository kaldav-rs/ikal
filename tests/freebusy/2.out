Ok(
    VFreebusy {
        dtstamp: Local(
            1997-09-01T12:00:00+02:00,
        ),
        uid: "19970901T095957Z-76A912@example.com",
        contact: None,
        dtstart: DateTime(
            Naive(
                1970-01-01T00:00:00,
            ),
        ),
        dtend: None,
        organizer: Some(
            "mailto:jane_doe@example.com",
        ),
        url: Some(
            "http://example.com/pub/busy/jpublic-01.ifb",
        ),
        attendee: [
            "mailto:john_public@example.com",
        ],
        comment: [
            "This iCalendar file contains busy time information for the next three months.",
        ],
        freebusy: [
            StartDur(
                StartDur {
                    start: Local(
                        1997-10-15T07:00:00+02:00,
                    ),
                    duration: Duration {
                        secs: 30600,
                        nanos: 0,
                    },
                },
            ),
            StartDur(
                StartDur {
                    start: Local(
                        1997-10-15T18:00:00+02:00,
                    ),
                    duration: Duration {
                        secs: 19800,
                        nanos: 0,
                    },
                },
            ),
            StartDur(
                StartDur {
                    start: Local(
                        1997-10-16T00:30:00+02:00,
                    ),
                    duration: Duration {
                        secs: 23400,
                        nanos: 0,
                    },
                },
            ),
        ],
        rstatus: [],
        x_prop: {},
        iana_prop: {},
    },
)
